Code/Framework/AzCore/AzCore/Jobs/Internal/JobManagerWorkStealing.h

```diff
@@ -15,18 +15,25 @@
 #include <AzCore/Memory/PoolAllocator.h>
 
 #include <AzCore/std/containers/queue.h>
+#include <AzCore/std/smart_ptr/unique_ptr.h>
+#include <AzCore/std/parallel/atomic.h>
 #include <AzCore/std/parallel/shared_mutex.h>
 #include <AzCore/std/parallel/mutex.h>
 #include <AzCore/std/parallel/semaphore.h>
 #include <AzCore/std/parallel/binary_semaphore.h>
 #include <AzCore/std/parallel/thread.h>
 
+// TODO: This is here to provide a fallback period if it's determined that the legacy workqueue behavior (unbounded growth and fine grained
+// prioritization is needed). Ping jeremyong-az if this define and the corresponding code is not removed by 2021/09
```

Comment:
Nitpick: Comments with dates and requests to call a specific person don't age well.